### DISCUSSION TITLE
addition to "tweaks" for normal width scroll bar?

### DISCUSSION THREAD

**Discussion by krc** [AprilSylph/XKit-Rewritten#1021] [Suggestion Box]
Tumblr set the scrollbar width to narrow, I find it really hard to use like that. I have a bookmarklet that works to make it wide, but I have to click it every time I go to tumblr or refresh.

This is the bookmarklet I came up with, seems to get set in two places, HTML and Body. Been using this for some weeks and works reliably. At first I just and one and it seemed to sometimes work? Not sure what was up with that. Anyway.

```
javascript: (() => {document.querySelector("html").style.setProperty('scrollbar-width', 'auto'); document.querySelector("body").style.setProperty('scrollbar-width', 'auto')})();
```

(I presume Xkit just has a style sheet it applies or something rather than JS like this)

**Comment by marcustyphoon** [comment]
You can set this permanently using [Stylus](https://addons.mozilla.org/en-US/firefox/addon/styl-us/)!:

```css
body, html {
    scrollbar-width: auto !important;
}
```

(Also, interesting; this property is Firefox-only!)

**Comment by krc** [comment]
Thanks!

Got it working with Firemonkey (same script probably also works for stylus etc), also added div to cover the blog pop-over view
In case anyone else goes looking for this kind of thing and ends up here:

```
/*
==UserCSS==
@name tumblr-wide-scrollbar
@author krc
@namespace https://github.com/krc/
@license CC0-1.0
@match https://*.tumblr.com/*
@version 0.20230309.3
@run-at document-end
==/UserCSS==
*/

body, html, div {
  scrollbar-width: auto !important;
}
```

**Comment by marcustyphoon** [comment]
Awesome!

I think this could be a reasonable addition to AccessKit, maybe? Kind of weird for it not to affect Chrome/Edge/Opera/etc, but I guess "in Firefox" isn't too long to add to the label.
